content/en/community/event/tid-episode-005/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: '05: Security in Istio'
+date: 2021-06-24T11:00:00+07:00
```

Comment:
You have to use a date before the PR was submitted, otherwise, the page won't render.